.travis.yml

```diff
@@ -48,11 +50,6 @@ jobs:
       before_script:
         - rustup component add rustfmt
       env: TASK=fmt-travis
-    - name: "lint Rust source using current development toolchain"
```

Comment:
We want to keep this...this and stable will still diverge w/ every update.

Reply:
I don't know if I follow what you mean.  Do we still want two lint tasks, but have them be the same until the update occurs, where one follows to the new stable version?

(Also, the task wasn't removed, it was just moved down to the "intermittently allowed" section, which I suppose will fold into the "allowed failures" section.)

Reply:
Also, there's no `env: TASK=clippy` task for stable.  Should there be?

Reply:
I did a little bit of restructuring; how does it look now?